conftest.py

```diff
@@ -75,6 +75,13 @@ def pytest_addoption(parser):
              "4KiB blocksize to be formatted and used in storage tests. "
              "Set it to 'auto' to let the fixtures auto-detect available disks."
     )
+    parser.addoption(
+        "--image-format",
+        action="append",
+        default=[],
+        help="Format of VDI to execute tests on."
+        "Example: vhd,qcow2"
```

Comment:
Making this a CLI parameter means you're deporting the test job definition outside pytest. When rishi wanted to do the same for thin vs thick in the context of XOSTOR tests, we asked him to rework the tests so they actually test both formats.

Reply:
It's to allow to test only one type by hand, the objectives for tests is to keep the default which at the moment is `vhd` but will be `["vhd", "qcow2"]` eventually.

Reply:
pytest already has test selection mechanisms:
* path to test files
* `-k`
* `-m`

Reply:
As long as the parameter is not necessary in `./jobs.py`, fine by me. We won't have SR types that only support VHD or only support QCOW2 at some point?

Reply:
The parameter default to VHD when not given meaning it's not needed to change jobs.py. But I will need to add the new coalesce tests in jobs.py either way and it's the only test at the moment that will use `vdi-type`
I'm not sure to see how the other selection mechanisms from pytest could work in this case.

Reply:
Works, I think we can move ahead with this merge.